pack-contracts/pack-contract-grpc/pom.xml

```diff
@@ -41,6 +41,11 @@
       <groupId>io.grpc</groupId>
       <artifactId>grpc-stub</artifactId>
     </dependency>
+    <dependency>
+      <groupId>com.google.protobuf</groupId>
+      <artifactId>protobuf-java</artifactId>
+      <version>3.19.2</version>
```

Comment:
I think we need to specify the dependency version here with protobuf-java.version.

Reply:
yes, I missed it here

Reply:
done